crates/extensions/c8y_mapper_ext/src/alarm_converter.rs

```diff
@@ -99,6 +92,70 @@ impl AlarmConverter {
         Ok(output_messages)
     }
 
+    pub(crate) fn new_try_convert_alarm(
+        &mut self,
+        parent_device_name: String,
+        input_message: &Message,
+    ) -> Result<Vec<Message>, ConversionError> {
+        let mut output_messages: Vec<Message> = Vec::new();
+        let alarm_id = input_message.topic.name.split('/').last();
```

Comment:
Since `alarm_id` should have a value to proceed the following step, the error should be handled in this line. Then stop using `expect` later as it will panic.

Reply:
fixed